client/src/components/new-request/search.vue

```diff
@@ -419,11 +422,8 @@ export default class NewSearch extends Vue {
   clearErrors () {
     newReqModule.clearErrors()
   }
-  displayRequestActionListItem (item) {
-    if (item && item.text) {
-      return item.text.replace('Relocate into', 'Relocate into BC')
-    }
-    return ''
+  setClass (value) {
+    return value ? 'disabled-custom' : ''
```

Comment:
Did you know you can have this logic in the `:class` property?

See
https://github.com/bcgov/namerequest/blob/1a25b0db8ee1389e89c7dd0392835d35706dd50a/client/src/components/tabs.vue#L15
or
https://github.com/bcgov/namerequest/blob/1a25b0db8ee1389e89c7dd0392835d35706dd50a/client/src/components/common/contact-info.vue#L2

Reply:
oooh cool okay I will update

Reply:
+1 for sexy code